diff --git a/CMakeLists.txt b/CMakeLists.txt
index 02c9235..8f2166c 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -23,7 +23,7 @@ project(iceoryx_posh VERSION ${IOX_VERSION_STRING})
 find_package(Threads REQUIRED)
 find_package(iceoryx_hoofs REQUIRED)
 
-option(DOWNLOAD_TOML_LIB "Download cpptoml via the CMake ExternalProject module" ON)
+option(DOWNLOAD_TOML_LIB "Download cpptoml via the CMake ExternalProject module" OFF)
 option(TOML_CONFIG "TOML support for RouDi with dynamic configuration" ON)
 option(ONE_TO_MANY_ONLY "Restricts communication to 1:n pattern" OFF)
 
